crates/core/tests/client.rs

```diff
@@ -1,22 +1,137 @@
+mod helpers;
+
 #[cfg(test)]
 mod tests {
 
     use std::str::FromStr;
 
+    use dojo_test_utils::sequencer::TestSequencer;
+    use ethers::types::{Address as EthersAddress, U256 as EthersU256};
     use kakarot_rpc_core::client::client_api::{KakarotEthApi, KakarotStarknetApi};
+    use kakarot_rpc_core::client::config::StarknetConfig;
     use kakarot_rpc_core::client::errors::EthApiError;
+    use kakarot_rpc_core::client::KakarotClient;
     use kakarot_rpc_core::mock::wiremock_utils::setup_mock_client_crate;
     use kakarot_rpc_core::models::block::BlockWithTxs;
     use kakarot_rpc_core::models::convertible::{ConvertibleStarknetBlock, ConvertibleStarknetEvent};
     use kakarot_rpc_core::models::event::StarknetEvent;
-    use kakarot_rpc_core::models::felt::Felt252WrapperError;
+    use kakarot_rpc_core::models::felt::{Felt252Wrapper, Felt252WrapperError};
     use kakarot_rpc_core::models::ConversionError;
     use reth_primitives::{Address, Bytes, H256};
     use reth_rpc_types::Log;
     use starknet::core::types::{BlockId, BlockTag, Event, FieldElement};
     use starknet::core::utils::get_selector_from_name;
     use starknet::providers::Provider;
 
+    use crate::helpers::constants::{EVM_ADDRESS, EVM_PRIVATE_KEY};
+    use crate::helpers::deploy_helpers::{create_raw_ethereum_tx, deploy_kakarot_system};
+
+    #[tokio::test]
+    #[ignore]
```

Comment:
why are we ignoring this test?

Reply:
The tests are ignored because the kakarot artifacts need to be moved to wherever we define `COMPILED_KAKAROT_PATH` in order to successfully deploy. I added the `KAKAROT_EVM_PROJECT_DIR` and the `scripts/copy_kakarot_deploy_artifacts.sh` with the thought that this can help the flow locally and also to do it in a github action before the tests run.

Wdyt? Follow me here? Perhaps there is a simpler option!